lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,92 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+
+        # Native options
+        ignore: one_or_more_strings?,
+      )
+    end
+
+    ALLOWED_ERRORS = [
+      "no project was found in any parent directories of",
+      "no YAML file was found in",
+    ].freeze
+
+    register_config_schema SCHEMA.config
+
+    def self.config_example
+      <<~'YAML'
+        root_dir: project/
+        target:
+          - .github/workflows/one_workflow.yml
+          - .github/workflows/another_workflow.yml
+        ignore:
+          - "job .* needs job"
+          - "unknown Webhook event"
+      YAML
+    end
+
+    # This analyser uses git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyzer_options
+      opts = []
+      opts << "-format"
+      opts << "{{json .}}"
+
+      Array(config_linter[:ignore]).each do |rule|
+        opts << "-ignore"
+        opts << rule
+      end
```

Comment:
This reduces lines a bit.

```suggestion
      Array(config_linter[:ignore]).each { |v| opts << "-ignore" << v }
```

Reply:
Nice, I didn't know about the repeated `<<` operator usage.